src/Entities/Interfaces/ISelection.java

```diff
@@ -4,12 +4,23 @@
 
 public interface ISelection {
 
-    float getPrice();
-
-    int size();
+    /**
+     * A method for getting the size of a selection
+     * @return the number of different addons a selection has
+     */
+    int getSize();
 
+    /**
+     * A method that gets the number of an addon in a selection
+     * @param addon the addon we want the amount for
+     * @return the number of the addon in the selection
+     */
     Integer get(IAddon addon);
```

Comment:
Perhaps we can rename get to getQuantityOfAddon? What do you think?

Reply:
Yeah I think that's a more descriptive name, I'll change it